data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -ex
+
+IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
+IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
+COREOS_DOWNLOADER_IMAGE=${COREOS_DOWNLOADER_IMAGE:-"quay.io/openshift-metal3/rhcos-downloader:master"}
+
+# This image is templated in via the installer pkg/asset/ignition/bootstrap/bootstrap.go
+RHCOS_IMAGE_BASE_URL="{{.BootImage}}"
+
+# First we stop any previously started containers, because ExecStop only runs when the ExecStart process
```

Comment:
It sounds like this comment wasn't finished - "only runs when the ExecStart process ..."

Reply:
The comment continues on the following line:

```
# First we stop any previously started containers, because ExecStop only runs when the ExecStart process
# e.g this script is still running, but we exit if *any* of the containers exits unexpectedly

```